RNA-seq/reproducibility_cmdline.md

```diff
@@ -46,18 +46,20 @@ This took a lot of time, and if we had typos that really confused things.
 Instead, we can write loops that work over many files.
 To perform QC over all of the fastq files in a directory we can write:
 ```bash
-mkdir QC
+mkdir <QC_DIRECTORY>
 
-for filename in fastq/*.fastq.gz
+for filename in <FASTQ_DIRECTORY>/*.fastq.gz
 do
   # remove leading path
   name=${filename##*/}
   # create directory for QC report
-  mkdir QC/${name} -p
+  mkdir <QC_DIRECTORY>/${name} -p
   # run fastqc
-  fastqc ${filename} -o QC/${name}
+  fastqc ${filename} -o <QC_DIRECTORY>/${name}
 done
 ```
+where `<QC_DIRECTORY> is the directory to hold the QC reports and <FASTQ_DIRECTORY> is the directory that contains the `fastq` files.
```

Comment:
Do you want to note that < and > mean put something here, and don't include `<` and `>`? This notation sometimes confuses people in our class.